plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemTrim.java

```diff
@@ -0,0 +1,79 @@
+package com.denizenscript.denizen.objects.properties.item;
+
+import com.denizenscript.denizen.objects.ItemTag;
+import com.denizenscript.denizen.utilities.Utilities;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.core.MapTag;
+import org.bukkit.Registry;
+import org.bukkit.inventory.meta.ArmorMeta;
+import org.bukkit.inventory.meta.trim.ArmorTrim;
+import org.bukkit.inventory.meta.trim.TrimMaterial;
+import org.bukkit.inventory.meta.trim.TrimPattern;
+
+public class ItemTrim extends ItemProperty<MapTag> {
+
+    // <--[property]
+    // @object ItemTag
+    // @name trim
+    // @input MapTag
+    // @description
+    // Controls the trim on armor.

```

Comment:
Can do something like `An armor item's trim` or whatever, so that the description makes sense for both the tag and the mech.
Also see review above; while they're armor only in vanilla, they can technically be changed to apply to other stuff as well - I'd probably document it as applying to any item and add a note about being armor only in default vanilla